dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/ConnectionPoolImpl.java

```diff
@@ -197,10 +197,10 @@ public boolean removeHost(Host host) {
 			cpHealthTracker.removeHost(host);
 			cpMonitor.hostRemoved(host);
 			hostPool.shutdown();
-            Logger.info(String.format("Remove host: Successfully removed host %s from connection pool", host.getHostName()));
+            Logger.info(String.format("Remove host: Successfully removed host %s from connection pool", host.getHostAddress()));
```

Comment:
Agreed. I just want to make sure that printing private ip address (which will be the case from now on) will not cause issues while debugging.